docs/docsgen/source/technical/int2.md

```diff
@@ -0,0 +1,40 @@
+<!--
+Copyright (c) ONNX Project Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+(onnx-detail-int2) = 
+
+
+# 2 bit integer types
+
+## Papers
+
+[T-MAC: CPU Renaissance via Table Lookup for Low-Bit LLM Deployment on Edge](https://arxiv.org/abs/2407.00088)
+
+T-MAC, an innovative lookup table(LUT)-based method designed for efficient low-bit LLM (i.e., weight-quantized LLM) inference on CPUs. T-MAC directly supports mpGEMM without dequantization, while simultaneously eliminating multiplications and reducing additions required. Specifically, T-MAC transforms the traditional data-type-centric multiplication to bit-wise table lookup, and enables a unified and scalable mpGEMM solution.
```

Comment:
If this is the primary motivation for 2-bit elements, then it would be nice to include a sentence describing and pointing to this in either the PR description or in https://github.com/onnx/onnx/issues/7159 (which is currently very sparse).